src/modules/apps/e2e/ErrorBoundary.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Color } from 'ink'
+
+export class ErrorBoundary extends React.Component<{}, { error: any }> {
+    constructor(props) {
+      super(props)
+      this.state = { error: null }
+    }
+  
+    public render() {
+      if (this.state.error != null) {
+        // You can render any custom fallback UI
+        return <Color red>{this.state.error.message}</Color>
+      }
+  
+      return this.props.children
+    }
+  }
```

Comment:
I'm starting to think that this doesn't have any effect :thinking:
Maybe test without it? If in the future want to add some improvements on error handling we can add something